### DISCUSSION TITLE
Add Star Repo to a  groups

### DISCUSSION THREAD

**Discussion by sanskritkm** [community/community#4352] [Mobile] [answered]
Example I have many Repo Star including learning like 30day HTML,and application project like Wordpress or PiVPN I want to group that into a group like Study Group, Network Group, or CMS Group.

**Comment by brianlovin** [comment, marked as answer]
This is a cool idea, thanks for sharing @williamgh2019 – I've added your feedback to a tracking issue we have internally about Stars and Star organization 😊
